0000-template.md

```diff
@@ -0,0 +1,26 @@
+# <Feature Name>
```

Comment:
Angle brackets seem not to work. Maybe you have to escape somehow?

Reply:
Done